packages/react/src/index.ts

```diff
@@ -8,8 +8,12 @@ import {
 	untracked,
 } from "@preact/signals-core";
 import type { ReactElement } from "react";
-import { useSignal, useComputed, useSignalEffect } from "../runtime";
-import { installAutoSignalTracking } from "../runtime/src/auto";
```

Comment:
This double import caused some duplicate code in the final bundle. Combining them is what caused this PR to reduce the overall bundle size shown in the size-bot comment.